Kotlin/search/LinearSearch.kt

```diff
@@ -0,0 +1,47 @@
+/*Linear search algorithm is an algorithm which checks if an element is present in the given list by comparing it with every element in the list.
+  Linear search is the simplest search algorithm;it is a special case of brute-force search and in worst-case, cost is proportional to the number
+  of elements in the list. Hence name is Linear search.This is a linear search algorithm in Kotlin.*/
+
+import java.util.*
```

Comment:
Why are you using this? I'm just genuinely curious about follow this [here](https://stackoverflow.com/questions/44852032/how-to-import-java-packages-in-kotlin-native)

Reply:
as we are using scanner obj we need it, I have just sent a picture to you on discord, it shows an error without importing